### ISSUE TITLE
BLK999 Unexpected exception: 'PY310'

### ISSUE THREAD

**Issue by Minipada** [peterjc/flake8-black#66]
# Setup
pyproject.toml contains:

```toml
[tool.black]
diff = false
verbose = false
line-length = 100
target-version = ['py38', 'py39', 'py310']
```

flake8-black version: 0.3.3
poetry version: 1.1.13

pre-commit:
```yaml
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.9.2
    hooks:
      - id: flake8
        additional_dependencies:
          [
            ...
            flake8-black,
            ...
          ]
```

# What I did
running:
```
poetry shell
flake8 ...
```
works

As does `poetry run flake8 ...`

But
```
poetry run pre-commit run flake8 ...
```

returns
```
BLK999 Unexpected exception: 'PY310'
```

Only this extension gives problems, not sure why.

**Comment by Minipada** [comment]
I think the error is triggered because some other extension interfers...If i remove `darglint` or `flake8-variables-names` in the the precommit configuration, this is not displayed anymore... I'm a bit lost on this one

**Comment by Minipada** [comment]
Before trying to edit the flake8_black.py, I cleaned up my pre-commit cache directory. After that, the error disappeared. I can't reproduce it at all now.
I will close the issue and if it comes back, I or other users could reproduce it.

**Comment by peterjc** [comment]
Curious - hopefully its won't come back. Thanks for the update.
